src/pyramid/config/routes.py

```diff
@@ -40,12 +40,9 @@ def add_route(
         inherit_slash=None,
         **predicates
     ):
-        """ Add a :term:`route configuration` to the current
-        configuration state, as well as possibly a :term:`view
-        configuration` to be used to specify a :term:`view callable`
-        that will be invoked when this route matches.  The arguments
-        to this method are divided into *predicate*, *non-predicate*,
-        and *view-related* types.  :term:`Route predicate` arguments
+        r""" Add a :term:`route configuration` to the current configuration state.
```

Comment:
I do not like this pattern of using an r-string so that you can make `**predicates` work. Is there some other way to escape those two characters like `\\*\\*` ?

Reply:
Worse case scenario I'd just use backticks to escape it.

```
``**predicates``
```

Reply:
That was one combination I did not try. Double-escaping works! `\\*\\*predicates`.